cmd/cli/cmd/typeinstance/get.go

```diff
@@ -158,3 +176,9 @@ func toTypeInstanceIDs(in []*gqllocalapi.TypeInstance) string {
 	}
 	return strings.Join(out, ", ")
 }
+
+func printHugePayloadWarning() {
+	warning := color.New(color.FgYellow).FprintfFunc()
+	warning(os.Stderr, "  Warning: ")
+	fmt.Fprintln(os.Stderr, "Fetching all TypeInstances with all fields. This may generate huge payload and request may be canceled due to timeout. Change '--timeout' value if needed.")
```

Comment:
```suggestion
	fmt.Fprintln(os.Stderr, "Fetching full details of all TypeInstances. This may take a while and generate huge output. Use '--timeout' flag to increase timeout if needed.")
```